airlock/templates/file_browser/group.html

```diff
@@ -7,7 +7,7 @@
     background-color: darkgrey;
   }
 </style>
-{% #card title=path_item.name container=True %}
+{% #card title=group.title|default_if_none:path_item.name container=True class="group_modal" %}
```

Comment:
in what context could `group.title` be none?

Reply:
ah, it was possibly none at an earlier point in the branch, when I added this. Will remove.

Reply:
fixed